processor/cumulativetodeltaprocessor/processor.go

```diff
@@ -150,7 +144,8 @@ func (ctdp *cumulativeToDeltaProcessor) shutdown(context.Context) error {
 }
 
 func (ctdp *cumulativeToDeltaProcessor) shouldConvertMetric(metric pmetric.Metric) bool {
-	return (ctdp.includeFS == nil || ctdp.includeFS.Matches(metric.Name())) &&
+	return (metric.AggregationTemporality() == pmetric.AggregationTemporalityCumulative) &&
```

Comment:
This doesn't work because AggregationTemporality is only defined for certain metrics (which is why were are doing it in the metric type switch statement).